### ISSUE TITLE
Color class is inconsistent about premultiplication

### ISSUE THREAD

**Issue by dtaylorus** [praeclarum/NGraphics#13]
new Color(r, g, b) is not the same as Color.FromRGB(rgb). This confused the heck out of me. It looks like there is some premultiplication going on in FromRGB but not in the ctors.

I wanted to add an alpha value to Color.FromRGB(), but there is no overload that takes an alpha value, so I switched to some of the ctors and found out the hard way that they don't premultiply. So I had to use Color.FromRGB(rgb).WithAlpha(a).

I suggest a naming convention that is more explicit about premultiplication, like the Monogame Framework Color.FromNonPremultiplied methods.

I also suggest overloads that take an alpha parameter, to make it simple to create a premultiplied color that includes an alpha value in one step.

Color.FromNonPremultiplied(int r, int g, int b, int a = 255)
Color.FromNonPremultiplied(int r, int g, int b, double a = 1.0)
Color.FromNonPremultiplied(int rgb, int a = 255)
Color.FromNonPremultiplied(int rgb, double a = 1.0)
Color.FromNonPremultiplied(uint argb)

**Comment by dtaylorus** [comment]
It might also be nice to have a Premultiply() method so you could do this:

new Color(r, g, b, a).Premultipy() as an alternative to the static method Color.FromNonPremultiplied(r, g, b, a)

**Comment by praeclarum** [comment]
Yeah FromRGB should be renamed FramPARGB or something (need to check the byte order). Sorry for the confusion!
